app/src/main/AndroidManifest.xml

```diff
@@ -45,6 +45,7 @@
         <activity
             android:name="org.p2p.wallet.root.RootActivity"
             android:exported="true"
+            android:hardwareAccelerated="false"
```

Comment:
are you sure to disable it for whole app?

Reply:
https://developer.android.com/topic/performance/hardware-accel#view-level
maybe only for view?